gitops/plnsvc/base/kcp-manager.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: plnsvc
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kcp-manager
```

Comment:
I am getting confused. The ServiceAccount name is kcp-manager but the name of the ClusterRole and its binding kcp-syncer and kcp-syncer-binding. What is that for?

Reply:
kcp-manager would be the SA that manages everything related to KCP. This currently only means the syncer, for which the appropriate role and binding are created.

If that's not a good model, let me know and I'll rename the SA to kcp-syncer.

Reply:
There should not be much more than the syncer agent. I think that it would be good that the names align in one way or the other. I would vote for having the SA named kcp-syncer.